catalog/ui/catalog-ui-search-api/src/main/java/org/codice/ddf/catalog/ui/subscription/SubscriptionConstants.java

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.catalog.ui.subscription;
+
+public class SubscriptionConstants {
+
+  private SubscriptionConstants() {}
+
+  public static final String SUBSCRIPTIONS_TYPE = "subscriptions";
+}
```

Comment:
❓ What is the benefit of creating a whole new class for one string constant vs. leaving it in `SubscriptionsPersistentStore`?

Reply:
@hayleynorton @Bdthomson 
There can be valid uses for both constants in the interface and in a separate file.  If the string is something that helps define the interface/contract then i would say it might belong in the interface. 

 If we look at this case closer, `SUBSCRIPTIONS_TYPE` is used  as the key of the type to store these objects under in the `PersistentStore`. There is also a `SUBSCRIPTIONS_TYPE` duplicated in `PersistentStore`. So some of the other persistent stores use the constants directly out of the persistent store. Since there's only one persistent store and technically consumers of it could step on each others toes if two services were to try and use the same key then we should probably just remove the key entirely from the `SubscriptionPersistentStore` and use the one from `PersistentStore`. People could still step on each others toes by using the same key, but at least using the ones in the `PersistentStore` means we have a single place to store all of these and people will have to see that there's one already in there before accidentally using it. 

So TL;DR- Maybe remove this constant and just use the `PersistentStore` constants inside the implementation of `SubscriptionPersistentStoreImpl`